testutils/ocispec.go

```diff
@@ -153,6 +153,33 @@ func AreSyscallsBlockedBySeccomp(seccompProfile specs.LinuxSeccomp, syscallNames
 	return true
 }
 
+// isSyscallWithArgsAllowedBySeccomp checks that the provided syscall and args are whitelisted by the seccomp profile
+func isSyscallWithArgsAllowedBySeccomp(seccompProfile specs.LinuxSeccomp, syscallName types.Syscall, syscallArgs []specs.LinuxSeccompArg) bool {
+	return !isSyscallWithArgsBlockedBySeccomp(seccompProfile, syscallName, syscallArgs)
+}
+
+// AreSeccompSyscallsWithArgsAllowed checks that the provided list of syscalls and args are whitelisted by the seccomp profile
+func AreSeccompSyscallsWithArgsAllowed(seccompProfile specs.LinuxSeccomp, syscallsWithArgs map[types.Syscall][]specs.LinuxSeccompArg) bool {
+	for syscallName, syscallArgs := range syscallsWithArgs {
+		if !isSyscallWithArgsAllowedBySeccomp(seccompProfile, syscallName, syscallArgs) {
+			return false
+		}
+	}
+
+	return true
+}
+
+// AreSyscallsAllowedBySeccomp checks that the provided syscalls are whitelisted by the seccomp profile
+func AreSyscallsAllowedBySeccomp(seccompProfile specs.LinuxSeccomp, syscallNames []types.Syscall) bool {
+	for _, syscallName := range syscallNames {
+		if !isSyscallWithArgsAllowedBySeccomp(seccompProfile, syscallName, []specs.LinuxSeccompArg{}) {
+			return false
```

Comment:
Can we test this?

Reply:
same question, inner or outer func?

Reply:
The case where the inner conditional (`if !isSyscallWithArgsAllowedBySeccomp(seccompProfile, syscallName, []specs.LinuxSeccompArg{})`) is true and `AreSyscallsAllowedBySeccomp` returns false.